config/application.rb

```diff
@@ -5,11 +5,11 @@
 require "active_model/railtie"
 require "active_job/railtie"
 require "active_record/railtie"
-require "active_storage/engine"
+# require "active_storage/engine"
 require "action_controller/railtie"
 require "action_mailer/railtie"
 require "action_view/railtie"
-require "action_cable/engine"
+# require "action_cable/engine"
 # require "sprockets/railtie"
```

Comment:
Do we still need those commented-out dependencies, or could they be removed?

Reply:
We will probably not need them. But if somebody wonders, why an expected Rails feature is missing, the cause is easier to spot with comments than when the entry is missing completely.